__tests__/app.test.js

```diff
@@ -109,4 +109,71 @@ expect(reviews).toBeSortedBy('created_at',  {
         )
     })
 });
+test('should return 400 when given an invalidId', () => {
+    return request(app)
+    .get("/api/reviews/chocolate")
+    .expect(400)
+    .then((({body:{msg}})=>{
+        expect(msg).toBe('Bad Request')
+        
+            }))
+});
+    })
+    
+
+describe('GET /api/reviews/:review_id/comments', () => {
+    test('should return an array of comments for a particular review', () => {
+        return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then(({body})=>{
+            const {comments} = body
+      
+            comments.forEach((comment)=>{
+                expect(comment).toEqual(
+                    expect.objectContaining({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at : expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        review_id : expect.any(Number),
+                    })
+                )
+            })
+        })
+    })
```

Comment:
Could you just add one more test here for a status 200, valid ID but has no comments? This should respond with an empty array of comments. 👍